discovery/pkg/apigee/provision.go

```diff
@@ -273,18 +273,20 @@ func (p provisioner) productModeCreateProduct(logger log.FieldLogger, targetProd
 		}...)
 
 		product = &models.ApiProduct{
-			ApiResources:  curProduct.ApiResources,
-			ApprovalType:  curProduct.ApprovalType,
-			Attributes:    attributes,
-			Description:   curProduct.Description,
-			DisplayName:   targetProductName,
-			Environments:  curProduct.Environments,
-			Name:          targetProductName,
-			Proxies:       curProduct.Proxies,
-			Quota:         quota,
-			QuotaInterval: quotaInterval,
-			QuotaTimeUnit: quotaTimeUnit,
-			Scopes:        curProduct.Scopes,
+			ApiResources: curProduct.ApiResources,
+			ApprovalType: curProduct.ApprovalType,
+			Attributes:   attributes,
+			Description:  curProduct.Description,
+			DisplayName:  targetProductName,
+			Environments: curProduct.Environments,
+			Name:         targetProductName,
+			Proxies:      curProduct.Proxies,
+			Scopes:       curProduct.Scopes,
+		}
+		if quota != "" {
```

Comment:
same as line 313, if quota is not empty, is there a chance that quotaInterval and quotaTimeout could be?

Reply:
Well, the quota had to be set for us to care about interval or timeout. If no quota is passed by the access request then none of the values will be set in the calling func.